src/core/server/metrics/logging/ecs.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * and the Server Side Public License, v 1; you may not use this file except in
+ * compliance with, at your election, the Elastic License or the Server Side
+ * Public License, v 1.
+ */
+
+/**
+ * Typings for some ECS fields which core uses internally.
+ * These are not a complete set of ECS typings and should not
+ * be used externally; the only types included here are ones
+ * currently used in core.
+ *
+ * @internal
+ */
+
+export interface EcsOpsMetricsEvent {
+  /**
+   * These typings were written as of ECS 1.7.0.
+   * Don't change this value without checking the rest
+   * of the types to conform to that ECS version.
+   *
+   * https://www.elastic.co/guide/en/ecs/1.7/index.html
+   */
+  ecs: { version: '1.7.0' };
+
+  // base fields
+  ['@timestamp']?: string;
+  labels?: Record<string, unknown>;
+  message?: string;
+  tags?: string[];
+  kind?: string;
+  category?: string[];
```

Comment:
The data that's logged is a combination of `os` and `host` metric data and, as such, I decided to indicate that in the `category` field.
These fields will only appear in the `meta` and not the log message.

Reply:
`kind` and `category` are [categorization fields](https://www.elastic.co/guide/en/ecs/current/ecs-category-field-values-reference.html#ecs-category-fields), belonging to the `event` namespace. I guess I was looking for ECS fields and got a little overeager here.

Reply:
Ahh, okay I missed those in the docs! Thanks for clarifying. Based on my understanding of the docs, I think your usage here makes sense then.

However, I think they should be grouped under `event` instead of alongside the base fields, correct?
```
event: {
  kind: 'metric',
  category: ['host', 'process'],
}
```
It looks like you could also optionally set [`event.type: 'info'`](https://www.elastic.co/guide/en/ecs/current/ecs-allowed-values-event-type.html#ecs-event-type-info)

Reply:
Done: https://github.com/elastic/kibana/pull/88070/files#diff-a6816f46cfcc7805f73421ab0097bed66ea7b99eaaa428190cafe3acbf331535R35 and https://github.com/elastic/kibana/pull/88070/files#diff-a6816f46cfcc7805f73421ab0097bed66ea7b99eaaa428190cafe3acbf331535R42-R45